src/nbb/impl/nrepl_server.cljs

```diff
@@ -227,5 +237,18 @@
             (when (fs/existsSync ".nrepl-port")
               (fs/unlinkSync ".nrepl-port")))))
 
-(defn init []
-  (start-server @nbb/opts))
+(defn
+  init
+  []
+  (let [eval-require (fn
+                       [ns-form]
+                       (when ns-form
```

Comment:
why is there a `when` here? if there is no ns-form, then the `(.then nil ...)` expression will probably crash.

Reply:
Yea true. I removed the `when`. I also was thinking if this should be a function (in `core.cljs`?).
Like `(defn eval-repl-requires []) ; promise`